etcd/files/etcd.service.jinja

```diff
@@ -0,0 +1,24 @@
+{% from "etcd/map.jinja" import etcd_settings with context -%}
+[Unit]
+Description=etcd {{ etcd_settings.install.version }} - highly-available key value store
+Documentation=https://github.com/coreos/etcd
+Documentation=man:etcd
+After=network.target
+Wants=network-online.target
+
+[Service]
+Environment=DAEMON_ARGS=
+Environment=ETCD_NAME=%H
+Environment=ETCD_DATA_DIR={{ etcd_settings.data_directory }}
+EnvironmentFile=-/etc/default/%p
+StandardOutput=syslog
+WorkingDirectory={{ etcd_settings.data_directory }}
+#ExecStart=/bin/sh -c "GOMAXPROCS=$(nproc) /usr/bin/etcd $DAEMON_ARGS"
```

Comment:
Why is there commented code here without a comment explaining why it's commented ?

Reply:
The recent push alerted me - This was so long ago that I no longer remember (and highlighting the reason to add a comment).  My guess is that it was an artifact I didn't intend for the PR but I will review and submit another PR to clean up.